function/parameter.go

```diff
@@ -7,17 +7,6 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/attr"
 )
 
-const (
-	// DefaultParameterNamePrefix is the prefix used to default the name of parameters which do not declare
-	// a name. Use this to prevent Terraform errors for missing names. This prefix is used with the parameter
-	// position in a function definition to create a unique name (param1, param2, etc.)
-	DefaultParameterNamePrefix = "param"
-
-	// DefaultVariadicParameterName is the default name given to a variadic parameter that does not declare
-	// a name. Use this to prevent Terraform errors for missing names.
-	DefaultVariadicParameterName = "varparam"
-)
```

Comment:
For breaking change completeness, we'll need to also call out these constant removals.

Reply:
Have added a changelog entry for this.